blog/2021-07-30-temporal-tips-tricks-1.md

```diff
@@ -0,0 +1,116 @@
+---
+tags:
+  - Temporal
+  - productivity
+  - tips
+posted_on_: 2021-07-30T00:00:09Z
+slug: temporal-tips-tricks-1
+title: 'Tips and Tricks for Temporal Developer Productivity'
+author: swyx
+author_title: Head of Developer Experience
+author_image_url: https://avatars.githubusercontent.com/u/6764957?v=4
+release_version: V1.11.2
+---
+
+
+<!--truncate-->
+
+Discussions about Temporal often get into very high minded distributed systems debates, but sometimes it can be helpful to swap small tips that can have a huge impact on developer productivity!
```

Comment:
```suggestion
Discussions about Temporal often get into very high-minded distributed systems debates, but sometimes it can be helpful to swap small tips that can have a huge impact on developer productivity!
```

Reply:
otherwise sounds like your mind is "high" ;)